### PR TITLE
support for fastly acls

### PR DESCRIPTION
**Pull request by kamsz**

### CODE REVIEW

File: fastly/resource_fastly_acl_v1.go [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+
+	if err != nil {
+		return err
+	}
+
+	acl, err := conn.CreateACL(&gofastly.CreateACLInput{
+		Name:    d.Get("name").(string),
+		Service: d.Get("service_id").(string),
+		Version: versionNumber,
+	})
```
Comment (ljagiello) [inline fastly/resource_fastly_acl_v1.go:81]: I think we should validate VCL before creation.

File: fastly/resource_fastly_acl_v1.go [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+		activateVersion(service.ID, versionNumber, meta)
+	}
+
+	d.SetId(acl.Name)
+	d.Set("version", acl.Version)
+	return resourceACLV1Read(d, meta)
+}
+
+func resourceACLV1Update(d *schema.ResourceData, meta interface{}) error {
+	conn := meta.(*FastlyClient).conn
```
Comment (ljagiello) [inline fastly/resource_fastly_acl_v1.go:97]: Same here, I think we should validate ACL before update.

**Comment by ljagiello** [comment]
Few general comments:
- I think you should validate ACL before any operations,
- If you want to push that to upstream (I hope you do) please add tests.

**Comment by kamsz** [comment]
Added validation before any ACL operation and tests.

**Pull request merged**